versions.yaml

```diff
@@ -21,43 +21,25 @@
 cloudbuild:
   enable_parallel: true
 versions:
+
 - dir: 3/debian10/3.8
   from: marketplace.gcr.io/google/debian10
   templateSubDir: default
```

Comment:
- remove `templates/3.7` directory as it is not used anymore
- move contents of `templates/default/*` to `templates/` and remove `templateSubDir: default`

Reply:
done